### PR TITLE
Update UI to show Appointments 

### LINKED ISSUE #74: Update GUI to show appointments for patients and doctors

### PR DESCRIPTION
**Pull request by bmanara**
Resolves #74

### CODE REVIEW

File: src/main/java/seedu/address/ui/DoctorCard.java [hunk 1]
```diff
@@ -61,5 +70,22 @@ public DoctorCard(Doctor doctor, int displayedIndex) {
         doctor.getTags().stream()
                 .sorted(Comparator.comparing(tag -> tag.tagName))
                 .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+
+        Set<Appointment> appointmentSet = doctor.getAppointments();
+        appointmentsLabel.setText("Appointments: " + appointmentSet.size());
+
+        ObservableList<String> appointmentStrings = FXCollections.observableArrayList(appointmentSet.stream()
```
Comment (chiangqinkang) [inline src/main/java/seedu/address/ui/DoctorCard.java:77]: Good use of streams

**Review by chiangqinkang — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/ui/DoctorCard.java [hunk 2]
```diff
@@ -61,5 +70,22 @@ public DoctorCard(Doctor doctor, int displayedIndex) {
...
+                        + appointment.getPatientName() + " on "
+                        + appointment.getDate().toString() + " (" + appointment.getTime().toString() + "hrs)")
+                .toArray(String[]::new));
+
+        appointments.setItems(appointmentStrings);
+
+        if (appointmentSet.isEmpty()) {
+            appointments.setPrefHeight(0);
+        } else {
+            appointments.setPrefHeight(50);
```
Comment (sam-theman88) [inline src/main/java/seedu/address/ui/DoctorCard.java:88]: Good catch to adjust height depending on size of set!

File: src/main/java/seedu/address/ui/PatientCard.java [hunk 3]
```diff
@@ -64,5 +73,22 @@ public PatientCard(Patient patient, int displayedIndex) {
...
                 .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+
+        Set<Appointment> appointmentSet = patient.getAppointments();
+        appointmentsLabel.setText("Appointments: " + appointmentSet.size());
+
+        ObservableList<String> appointmentStrings = FXCollections.observableArrayList(appointmentSet.stream()
+                .map(appointment -> "Appointment " + appointment.getId() + ": With Dr. "
+                        + appointment.getDoctorName() + " on "
+                        + appointment.getDate().toString() + " (" + appointment.getTime().toString() + "hrs)")
+                .toArray(String[]::new));
```
Comment (sam-theman88) [inline src/main/java/seedu/address/ui/PatientCard.java:84]: Great use of syntactic sugar to make code neater, more readable, and easier to understand:)

**Review by sam-theman88 — APPROVED** [review]
Overall LGTM! Detailed and quality work!

**Pull request merged**